src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocparamorder/InputJavadocParamOrderIncorrect.java

```diff
@@ -0,0 +1,115 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocparamorder;
+
+/**
+ * Config: default
+ * Some Javadoc.
+ *
+ * @since something
+ */
+class InputJavadocParamOrderIncorrect {
+
+    /**
+     * Description.
+     *
+     * @version Some javadoc. // OK
+     * @param p2
+     *            Parameter 2 desc. // violation
+     * @param p1
+     *            Parameter 1 desc.
+     * @param <T>
+     *            Parameter T desc. // violation
+     * @see Some javadoc. // OK
+     * @param p3
+     *            Parameter 3 desc.
+     * @return Some number. // OK
+     */
+    static <T> int test(String p1, String p2, T p3) { // violation
+        return 2021;
+    }
+
+    /**
+     * Description
+     *
+     * @param <i>
+     *            Parameter i desc. // violation
+     * @param str
+     *            Parameter str desc.
+     */
+    void test2(int i, String str) { // violation
+    }
+
+    /**
+     * Description.
+     *
+     * @version Some javadoc. // OK
+     * @param <E>
+     *            Parameter E desc. // violation
+     * @param <E>
+     *            Parameter E desc.
```

Comment:
This is a good case (duplicate tags); I am not sure how we should handle this, but I would think that the violation should be logged on the second ( and all consecutive) occurrences. Please add more than two duplicates, and mix them in with other `@param` tags in other javadocs, and let's check the behavior out.

Reply:
> that the violation should be logged on the second

but it seems correct to me bcoz in javadoc 1st param tag should be of <T> but its not so violation. 

> Please add more than two duplicates

it will work bcoz it will check one tag  in javadoc and list of param for that method of record whatever at a time if they are not equal it will give vioaltion.

>mix them in with other @param tags in other javadocs

i didnt get this

Reply:
> that the violation should be logged on the second

but it seems correct to me bcoz in javadoc 1st param tag should be of `<T>` but its not, so violation. 

> Please add more than two duplicates

it will work bcoz it will check one tag  in javadoc and list of param for that method or record whatever at a time if they are not equal it will give vioaltion.

>mix them in with other @param tags in other javadocs

i didnt get this

but i am searching more tests cases .

Reply:
Let's go with suggestion at https://github.com/checkstyle/checkstyle/issues/8990#issuecomment-829374503. We will ignore sequential duplicates, and only log duplicates that are not sequential.

Reply:
done

Reply:
Resolved.